### PR TITLE
reduce table shifting, add avatar templates

### PR DESCRIPTION
**Pull request by josh-hemphill**
I was going through to fix my profile image in the README.md, and realized there wasn't any consistency to the avatar icons and all look a little tacky if someone is using github's dark theme since there's no transparency around the circle. So I created an svg template with a clip path that you can load up in most tools that support svg and export with transparency.
I also added SVGs for the icons used in committee section, since those also look kind of funky in in other themes.
While messing with things I discovered that adding `min-width` to the contributor template somehow still affects the profile images even though it gets sanitized away, so I added that so that there's a lot less shifting of the contributors table.

I probably need to revert it, but I was curious if there's any reason the README's for the other languages aren't included in the automated contributors updates?

### CODE REVIEW

File: assets/images/members/template-collaborators.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg">
+	<clipPath id="clipCircle">
+		<circle r="100" cx="100" cy="100" />
+	</clipPath>
+	<image clip-path="url(#clipCircle)" width="200" height="200"
+		href="https://avatars.githubusercontent.com/goldbergyoni">
+	</image>
+	<circle r="97" cx="100" cy="100" stroke="#0efeff" stroke-width="6" fill="none" />
+</svg>
```
Comment (elektronik2k5) [inline assets/images/members/template-collaborators.svg:9]: This image looks broken in the PR preview.

File: assets/images/members/template-committee.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<clipPath id="clipCircle">
+		<circle r="100" cx="100" cy="100" />
+	</clipPath>
+	<image clip-path="url(#clipCircle)" width="200" height="200"
+		href="https://avatars.githubusercontent.com/goldbergyoni"
+		xlink:href="https://avatars.githubusercontent.com/goldbergyoni">
+	</image>
+	<circle r="97" cx="100" cy="100" stroke="#33cd32" stroke-width="8" fill="none" />
+</svg>
```
Comment (elektronik2k5) [inline assets/images/members/template-committee.svg:10]: This one looks broken too.

**Review by elektronik2k5 — COMMENTED** [review]
LGTM, except for the two images I commented on.

**Comment by josh-hemphill** [comment]
I suppose I should add a markdown file with instructions.
Those are just for creating new images for the collaborators and committee sections. They can't be used in markdown since github's CSP/CORS headers prevent the linked image inside an SVG from loading. So people can just change the username in it, and load it in any image editing program that supports SVG, and export your PNG with transparency. I tested it in GIMP and Inkscape.

**Comment by josh-hemphill** [comment]
Just replaced the template SVGs with a guide with the template SVG inlined for people to copy into a text file.

**Review by elektronik2k5 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1508,18 +1508,18 @@ Full Stack Developer & Site Reliability Engineer based in New Zealand, intereste
...
 
 [Josh Hemphill](https://github.com/josh-hemphill)
-<a href="https://twitter.com/spooklogical"><img src="assets/images/twitter-s.png" width="16" height="16"></img></a>
-<a href="https://www.linkedin.com/in/joshuahemphill/"><img src="assets/images/linkedin.png" width="16" height="16"></img></a>
-<a href="https://joshuahemphill.com"><img src="assets/images/www.png" width="16" height="16"></img></a>
+<a href="https://twitter.com/spooklogical"><img src="assets/images/twitter.svg" width="16" height="16"></img></a>
+<a href="https://www.linkedin.com/in/joshuahemphill/"><img src="assets/images/linkedin.svg" width="16" height="16"></img></a>
+<a href="https://joshuahemphill.com"><img src="assets/images/web.svg" width="16" height="16"></img></a>
 
 Full Stack Software Engineer / Developer specializing in Security, DevOps/DevSecOps, and ERP Integrations.
```
Comment (rluvaton) [inline README.md:1525]: It looks like you forgot to update my row 😉

**Review by rluvaton — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (goldbergyoni) [reply README.md:1525]: @josh-hemphill I did not disable it intentionally

**Review by elektronik2k5 — APPROVED** [review]

File: .all-contributorsrc [hunk 4]
```diff
@@ -1,9 +1,19 @@
 {
   "files": [
-    "README.md"
+    "README.basque.md",
```
Comment (rluvaton) [inline .all-contributorsrc:3]: Sorry for not saying this on the first review...
Can we use some sort of glob:RegEx here do we won't need to update this every time, something like `/`/README.*.md`

**Review by rluvaton — APPROVED** [review]

File: .all-contributorsrc (hunk 4 above)
Comment (josh-hemphill) [reply .all-contributorsrc:3]: I think I tried that with the `all-contributors` cli, and it didn't seem to support any kind of globbing.
A github action could probably be used in conjunction with the cli to inherit the bash expansion of glob patterns, but that would take a while longer to figure out if it's viable.

Comment (rluvaton) [reply .all-contributorsrc:3]: So that's fine for now!m, Great work

**Comment by goldbergyoni** [comment]
@josh-hemphill From my perspective, nothing prevents from merging. Your call

**Pull request merged**